### PR TITLE
Adding sponsors page

### PR DESCRIPTION
**Pull request by svetliomihailov**

### CODE REVIEW

File: source/index.html.erb [hunk 1]
```diff
@@ -106,6 +106,26 @@ title: Homepage
   </div>
 </div>
 
+<div class="spacing-both">
+  <div class="container accent-container">
+    <h3>Our partners</h3>
```
Comment (gsamokovarov) [inline source/index.html.erb]: `h2`

File: source/index.html.erb [hunk 2]
```diff
@@ -106,6 +106,26 @@ title: Homepage
...
+
+    <div class='index-sponsors-section'>
+      <%= link_to 'Learn more', '/partners', class: 'btn-primary' %>
+    </div>
+  </div>
+</div>
+
 <div class="spacing-both light" id="sponsorship">
   <div class="container">
     <h3>Call for sponsors</h3>
```
Comment (gsamokovarov) [inline source/index.html.erb]: `h2`

File: source/partners.html.erb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: Partners
+---
+
+<%= partial 'header' %>
+
+<div class="section-title spacing-both">
+  <div class="container">
+    <h1>Thanks to our Partners</h1>
+    <hr>
```
Comment (gsamokovarov) [inline source/partners.html.erb]: Do we really need it?

File: source/index.html.erb (hunk 1 above)
Comment (svetliomihailov) [inline source/index.html.erb]: ah ok i went with `call for sponsors`'s size which is h3 i will change it too.

File: source/partners.html.erb (hunk 3 above)
Comment (svetliomihailov) [inline source/partners.html.erb]: we can do without it.

**Pull request merged**